app/javascript/mastodon/features/ui/components/image_loader.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import ImmutablePureComponent from 'react-immutable-pure-component';
+import PropTypes from 'prop-types';
+
+class ImageLoader extends ImmutablePureComponent {
```

Comment:
OK I read https://facebook.github.io/react/docs/react-api.html#react.purecomponent and so I think I understand. It's fine to change.